src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php

```diff
@@ -27,11 +27,13 @@
  */
 class ImportMapRenderer
 {
+    private const DEFAULT_ES_MODULE_SHIMS_POLYFILL_URL = 'https://ga.jspm.io/npm:es-module-shims@1.8.0/dist/es-module-shims.js';
```

Comment:
Do we have another option ? Maybe in the recipe ? 
This package seem to release versions quite often.. https://www.npmjs.com/package/es-module-shims?activeTab=versions

Reply:
The thinking is that:

A) By default, the user does nothing and has no config and we give them this CDN URL. We can bump this URL when needed in core (other than if they have a major new version - but we can cross that bridge then).

B) This string itself doesn't need to be configurable because, if you care enough to fix it at a specific version, you'd add it to your importmap.

So I don't want to make this string configurable or put it in the recipe because then there would be 2 ways to control it - i.e. you'd be able to configure the fallback CDN URL and also take full control over it by putting it in your importmap.

Reply:
That make sense... maybe then just create a "monthly" reminder to check the new versions :)

Reply:
I think people will make PR's if they care enough. There's not a lot happening with that polyfill at this point, as it's quite feature complete.